src/sites/twitch-twilight/modules/directory/index.js

```diff
@@ -0,0 +1,205 @@
+		this.ChannelCard.on('unmount', inst => this.parent.clearUptime(inst), this);
+		this.css_tweaks.toggleHide('profile-hover-game', this.settings.get('directory.following.show-channel-avatar') === 2);
+	}
+
+
+	modifyStreams(res) { // eslint-disable-line class-methods-use-this
+		const nodeType = res.data.directory.__typename;
+		const newStreams = [];
+
+		const hiddenThumbnails = this.settings.provider.get('directory.game.hidden-thumbnails') || [];
+
+		const edges = res.data.directory.streams.edges;
+		for (let i = 0; i < edges.length; i++) {
+			const edge = edges[i];
+			const node = edge.node;
+
+			const s = node.viewersCount = new Number(node.viewersCount || 0);
+			s.profileImageURL = node.broadcaster.profileImageURL;
+			s.createdAt = node.createdAt;
+
+			if (nodeType === 'Game' && hiddenThumbnails.includes(res.data.directory.displayName) ||
+			nodeType === 'Community' && hiddenThumbnails.includes(node.game.name))
+				edge.node.previewImageURL = 'https://static-cdn.jtvnw.net/ttv-static/404_preview-320x180.jpg';
+
+			newStreams.push(edge);
+		}
+		res.data.directory.streams.edges = newStreams;
+		return res;
+	}
+
+
+	clearUptime(inst) { // eslint-disable-line class-methods-use-this
+		if ( inst.ffz_update_timer ) {
+			clearInterval(inst.ffz_update_timer);
+			inst.ffz_update_timer = null;
+		}
+
+		if ( inst.ffz_uptime_el ) {
+			inst.ffz_uptime_el.parentElement.removeChild(inst.ffz_uptime_el);
+			inst.ffz_uptime_el = null;
+			inst.ffz_uptime_span = null;
+			inst.ffz_uptime_tt = null;
+		}
+	}
+
+
+	updateUptime(inst, created_path, selector) {
+		const container = this.fine.getHostNode(inst),
+			card = container && container.querySelector && container.querySelector(selector),
+			setting = this.settings.get('directory.following.uptime'),
+			created_at = get(created_path, inst),
+			up_since = created_at && new Date(created_at),
+			uptime = up_since && Math.floor((Date.now() - up_since) / 1000) || 0;
+
+		if ( ! card || setting === 0 || uptime < 1 )
+			return this.clearUptime(inst);
+
+		const up_text = duration_to_string(uptime, false, false, false, setting === 1);
+
+		if ( ! inst.ffz_uptime_el )
+			card.appendChild(inst.ffz_uptime_el = e('div',
+				'video-preview-card__preview-overlay-stat c-background-overlay c-text-overlay font-size-6 top-0 right-0 z-default inline-flex absolute mg-05',
+				e('div', 'tw-tooltip-wrapper inline-flex', [
+					e('div', 'tw-stat', [
+						e('span', 'c-text-live tw-stat__icon', e('figure', 'ffz-i-clock')),
+						inst.ffz_uptime_span = e('span', 'tw-stat__value')
+					]),
+					inst.ffz_uptime_tt = e('div', 'tw-tooltip tw-tooltip--down tw-tooltip--align-center')
+				])));
+
+		if ( ! inst.ffz_update_timer )
+			inst.ffz_update_timer = setInterval(this.updateUptime.bind(this, inst, created_path, selector), 1000);
+
+		inst.ffz_uptime_span.textContent = up_text;
+		inst.ffz_uptime_tt.textContent = up_since.toLocaleString();
+
+	}
+
+
+	addCardAvatar(inst, created_path, selector) {
```

Comment:
This method does not make use of `created_path`, so this variable should be removed.